config.ts

```diff
@@ -0,0 +1,72 @@
+import { Address, dataSource, log } from '@graphprotocol/graph-ts'
+// import { address as ganacheAddress } from './config/ganache.json'
+// import { address as mainnetAddress } from './config/mainnet'
+// import { address as rinkebyAddress } from './config/rinkeby'
+
+export class TokenDetails {
+  name: string
+  symbol: string
+  address: Address
+}
+class TokenDetailsByNetwork {
+  mainnet: TokenDetails[]
+  rinkeby: TokenDetails[]
+  ganache: TokenDetails[]
+}
+
+// FIXME: Find a nicer solution. I cannot load from JSON ('./config/ganache.json')
+const GANACHE_NETWORK = '0x9561C133DD8580860B6b7E504bC5Aa500f0f06a7'
+const RINKEBY_NETWORK = '0xC576eA7bd102F7E476368a5E98FA455d1Ea34dE2'
+/**
+ * List of deprecated tokens. They are renamed to avoid users to get confused with this token, and the newer version
+ */
+let deprecatedTokensByNetwork: TokenDetailsByNetwork = {
+  mainnet: [
+    {
+      name: 'Synth sUSD (deprecated)',
+      symbol: 'sUSD-old',
+      address: Address.fromString('0x57Ab1E02fEE23774580C119740129eAC7081e9D3'),
+    },
+    {
+      name: 'Synthetix Network Token (deprecated)',
+      symbol: 'SNX-old',
+      address: Address.fromString('0xC011A72400E58ecD99Ee497CF89E3775d4bd732F'),
+    },
+  ],
+  rinkeby: [
+    {
+      name: 'Synth sUSD (deprecated)',
+      symbol: 'sUSD-old',
+      address: Address.fromString('0x1b642a124cdfa1e5835276a6ddaa6cfc4b35d52c'),
+    },
```

Comment:
So there's a deprecated one on rinkeby, is there a deprecated sUSD too?
Mind adding them in dex-react in config for `disabledTokens.rinkeby`

Reply:
There's no that I'm aware, actually. I just wanted to test it. I could use another contract for this actually.
I just want to override one address in Rinkeby